docs/content/node/guides/roles/getting-started.md

```diff
@@ -0,0 +1,244 @@
+---
+title: Getting started
+weight: 2
+description: >
+    Get started with Oso Roles
+---
+
+# Getting started
+
+When managing access to resources within an application, it can be useful to
+group permissions into **roles**, and assign these roles to users. This is
+known as **Role-Based Access Control (RBAC).** The `Oso` core library
+comes with built-in configuration for role-based access control.
+
+In this guide, we'll walk through the basics of starting to use the
+roles feature.
+
+## Setting up the Oso Instance
+
+First, we'll cover some of the basics of integrating Oso into your
+application.
+
+The `oso.Oso` class is the entrypoint to using Oso in our application.
+We usually will have a global instance that is created
+during application initialization and shared across requests. In a Flask
+application, you may attach this instance to the global flask object, or the
+current application if it needs to be accessed outside of the application
+initialization process.
```

Comment:
Flask mentioned in Node.js version of the doc — we should update the Node doc to mirror the recent updates made to the Python doc